app/scripts/controllers/createFromURL.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:CreateFromURLController
+ * @description
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('CreateFromURLController', function ($scope, $routeParams, $location, $filter, AuthService, DataService, AlertMessageService ) {
+    AuthService.withUser();
+
+    AlertMessageService.getAlerts().forEach(function(alert) {
+      $scope.alerts[alert.name] = alert.data;
+    });
+    AlertMessageService.clearAlerts();
+
+    $scope.alerts = {};
+    $scope.selected = {};
+
+    var messages = {
+      inaccessibleNamespace: _.template('You do not have access to the namespace "<%= namespace %>".'),
+      invalidImageStream: _.template('The requested image stream "<%= imageStream %>" is not valid.'),
+      invalidImageTag: _.template('The requested image stream tag "<%= tag %>" is not valid.'),
+      invalidNamespace: _.template('Resources from the namespace "<%= namespace %>" are not permitted.'),
+      invalidResource:'Image streams and templates cannot be combined.',
+      invalidTemplate: _.template('The requested template "<%= template %>" is not valid.'),
+      resourceRequired: 'A valid image stream or template is required.'
+    };
```

Comment:
Some of these are templates, and some are just strings. I'm not sure I like mixing them in the same object because you have no way to know which is which where they're called. Is this really better than just inlining the messages? Then there's no confusion.